appveyor.yml

```diff
@@ -29,6 +31,7 @@ init:
 # Build
 
 install:
+  - choco install vmwareworkstation # to have vmrun.exe for unit tests
```

Comment:
This is just a try for your go unit tests. I have seen that it searches for `vmrun.exe`, so why not installing VMware Workstation and test against a real installation?
But this slows down the build and test at AppVeyor.